scm/scm.go

```diff
@@ -214,3 +219,8 @@ type Team struct {
 type Authorization struct {
 	Scopes []string
 }
+
+// RepositoryInvitation represents a repository invitation.
```

Comment:
Suggest to rename this:
```go
// RepositoryInvitation represents a repository invitation.
type RepositoryInvitation struct {
	ID   int64
	Repo string
}
```
Note also that it is better to use `int64` here to avoid type casting; since it is only used in this one place anyway. (I don't recall why we kept these IDs as `uint64`)
Also, I added the `Repo` field to allow printing the error message... but it does not seem to be necessary for the logic of the function.